web/blob/static/js/graph.js

```diff
@@ -61,6 +63,18 @@ Prometheus.Graph.prototype.initialize = function() {
       e.preventDefault();
     }
 
+    if (e.which == 74 && e.shiftKey && self.expr.curElement > 0) {
```

Comment:
Can you extract the magic number into a "constant" please (however to solve that best in javascript). A clear name describing the key helps other readers to understand what's going on.